docs/templating/filters.md

```diff
@@ -0,0 +1,222 @@
+# Jinja2 Filters
+
+Peering Manager exposes functions and filters in additional to existing Jinja2
+provided ones. These filters are used to parse, transform, fetch values of
+known types. If they are not used as expected, template processing may result
+in failure or half rendered texts.
+
+### `safe_string`
+
+Converts a string to another one using only safe characters (retaining only
+ASCII characters). This string should be usable in a configuration without
+encoding issues.
+
+Example:
+```
+description "Peering: AS{{ a_s.asn }} {{ a_s.name | safe_string }}"
+```
+
+### `tags`
+
+For any objects having tags, it will give back only tags in an iterable
```

Comment:
I'd probably change this to say something like:

"Returns an iterable structure for all tags assigned to an object"